ContactDetailsApi/V1/UseCase/CreateContactUseCase.cs

```diff
@@ -22,22 +24,17 @@ public CreateContactUseCase(IContactDetailsGateway gateway, ISnsGateway snsGatew
             _snsFactory = snsFactory;
         }
 
-        public async Task<ContactDetailsResponseObject> ExecuteAsync(ContactDetailsRequestObject contactRequest, Token token)
+        public async Task<ContactDetailsResponseObject> ExecuteAsync(ContactDetailsRequestObject contactRequest,
+            Token token, string eventType)
```

Comment:
Why pass the event type in? This use case is specific already...

Reply:
Because that's the way I like it ! (But even so, fixed it :P )